rpmlint/descriptions/PAMModulesCheck.toml

```diff
@@ -0,0 +1,5 @@
+pam-unauthorized-module="""
+The package installs a PAM module. If the package
+is intended for inclusion in any SUSE product please open a bug
+report to request review of the service by the security team.
+Please refer to https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs"""
```

Comment:
Nope. This message is not okay. Please change this to something more generic.

Reply:
The idea was to put the patch to our `opensuse` branch. Is the PR acceptable to master with the adjusted description?

Reply:
Yes, provided this check is disabled by default. I expect anyone enabling this check to override the description and set up a list of allowed PAM modules.